test/Sdk.Generator.Tests/FunctionMetadataProviderGeneratorTests/DiagnosticResultTests.cs

```diff
@@ -193,6 +193,44 @@ public class MultiReturnHttp
                     expectedOutput,
                     expectedDiagnosticResults);
             }
+
+            [Fact]
+            public async void InvalidRetryOptionsFailure()
+            {
+                var inputCode = @"using System;
+                using System.Threading.Tasks;
+                using Microsoft.Azure.Functions.Worker;
+                using Microsoft.Azure.Functions.Worker.Http;
+
+                namespace FunctionApp
+                {
+                    public class HttpFunction
+                    {
+                        [Function(""HttpFunction"")]
+                        [FixedDelayRetry(5, ""00:00:10"")]
```

Comment:
Do we have a test for the case where the value of the attribute is in incorrect format?
Ex: ` [FixedDelayRetry(5, "10")]`

Reply:
While looking into this, realized there is an unanswered design question we should figure out: https://github.com/Azure/azure-functions-dotnet-worker/issues/1694. Resolving this comment and will tackle it after this item is discussed.